packages/fluentui/react-bindings/src/hooks/useIFrameFocusDispatch.ts

```diff
@@ -8,22 +8,30 @@ export const FUI_FRAME_EVENT = 'fuiframefocus';
 export const useIFrameFocusDispatch = (
   enableFrameFocusDispatch: boolean,
   targetDocument: Document,
-  pollDuration: number = 1000,
+  pollDuration: number = 100,
 ) => {
-  const timeoutRef = React.useRef<number>();
-
-  React.useEffect(() => {
-    if (enableFrameFocusDispatch) {
-      timeoutRef.current = targetDocument?.defaultView?.setInterval(() => {
-        const activeElement = targetDocument?.activeElement;
-        if (activeElement?.tagName === 'IFRAME' || activeElement?.tagName === 'WEBVIEW') {
-          const event = new CustomEvent(FUI_FRAME_EVENT, { bubbles: true });
-          activeElement.dispatchEvent(event);
-        }
-      }, pollDuration);
+  React.useEffect((): (() => void) | void => {
+    if (!enableFrameFocusDispatch || !targetDocument) {
```

Comment:
```suggestion
    if (!enableFrameFocusDispatch) {
```

<img width="550" alt="image" src="https://github.com/user-attachments/assets/6297a0f2-0f86-459c-ad87-cea6731a0e13" />


This does not match typings

Reply:
Thanks
This is a 1:1 convertion from the original code, where the usage was like `targetDocument?.defaultView`
I'll just go through the usages and remove unnecessary `null/undefine` check